### ISSUE TITLE
Entire Cifs share wiped when container is removed

### ISSUE THREAD

**Issue by dreh23** [ContainX/docker-volume-netshare#124]
I am scheduling several container with hashicorp's nomad. After I recreate the nomad job my share is wiped. Maybe a cifs version of #104 

```
root@cell-core-01:~#docker-volume-netshare version
Version: 0.33 - Built: 2017-01-08T22:45:48-08:00
```

```
root@cell-core-01:~# docker volume inspect  myserver.my.domain/share
[
    {
        "Driver": "cifs",
        "Labels": null,
        "Mountpoint": "/var/lib/docker-volumes/netshare/cifs/myserver.my.domain/share",
        "Name": "myserver.my.domain/share",
        "Options": {},
        "Scope": "local"
    }
]
```

```
root@cell-core-01:~# docker version
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Mon Mar 27 17:14:09 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Mon Mar 27 17:14:09 2017
 OS/Arch:      linux/amd64
 Experimental: false
```
cat nomad.job

```
...
        volumes = [
            "myserver.my.domain/share:/export"
        ]
        volume_driver = "cifs"
...
```

**Comment by dreh23** [comment]
Probably duplicate of: #122

**Comment by gondor** [comment]
This has been merged and released under version 0.34
